packages/core-contracts/contracts/ownership/Ownable.sol

```diff
@@ -24,22 +25,22 @@ contract Ownable is IOwnable, OwnableMixin, CommonErrors {
         OwnableStore storage store = _ownableStore();
 
         if (newNominatedOwner == address(0)) {
-            revert InvalidAddress(newNominatedOwner);
+            revert AddressError.ZeroAddress();
         }
 
         if (newNominatedOwner == store.nominatedOwner) {
-            revert InvalidNomination(newNominatedOwner);
+            revert ChangeError.NoChange();
         }
 
         store.nominatedOwner = newNominatedOwner;
         emit OwnerNominated(newNominatedOwner);
     }
 
-    function renounceNomination() external override onlyOwner {
```

Comment:
@i-stam I think I found an actual problem in the logic here.

The code that was in place semantically represents `removeNomination`, which the owner would be able to do.

`renounceNomination` should be done by whoever is nominated.